tests/vmi_configuration_test.go

```diff
@@ -2212,18 +2212,24 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 		})
 
 		It("[test_id:5360]should set appropriate IO modes", func() {
-			vmi := libvmifact.NewCirros(
+			libstorage.CreateHostPathPv("alpine-host-path", testsuite.GetTestNamespace(nil), testsuite.HostPathAlpine)
```

Comment:
Isn't `tests.DiskAlpineHostPath` a constant for `alpine-host-path`?

Reply:
Does this need to be cleaned up?

Reply:
It's done in https://github.com/kubevirt/kubevirt/blob/6f01c5ffcb2bd01f8deba622f3ef78c9deea0f4f/tests/testsuite/namespace.go#L190-L191

Reply:
> Isn't `tests.DiskAlpineHostPath` a constant for `alpine-host-path`?

Nope